app/controllers/catalog_controller.rb

```diff
@@ -69,26 +69,27 @@ class CatalogController < ApplicationController
     # * If left unset, then all facet values returned by solr will be displayed.
     # * If set to an integer, then "f.somefield.facet.limit" will be added to
     # solr request, with actual solr request being +1 your configured limit --
-    # you configure the number of items you actually want _displayed_ in a page.    
+    # you configure the number of items you actually want _displayed_ in a page.
     # * If set to 'true', then no additional parameters will be sent to solr,
     # but any 'sniffed' request limit parameters will be used for paging, with
-    # paging at requested limit -1. Can sniff from facet.limit or 
+    # paging at requested limit -1. Can sniff from facet.limit or
     # f.specific_field.facet.limit solr request params. This 'true' config
     # can be used if you set limits in :default_solr_params, or as defaults
     # on the solr side in the request handler itself. Request handler defaults
     # sniffing requires solr requests to be made with "echoParams=all", for
-    # app code to actually have it echo'd back to see it.  
+    # app code to actually have it echo'd back to see it.
     #
-    # :show may be set to false if you don't want the facet to be drawn in the 
+    # :show may be set to false if you don't want the facet to be drawn in the
     # facet bar
     config.add_facet_field 'format_main_ssim', label: 'Resource type'
-    config.add_facet_field 'pub_date', label: 'Date'  
+    config.add_facet_field 'pub_date', label: 'Date'
     config.add_facet_field 'language', label: 'Language'
     config.add_facet_field 'author_person_facet', label: 'Author', limit: true
     config.add_facet_field 'topic_facet', label: 'Topic', limit: true
     config.add_facet_field 'geographic_facet', label: 'Region', limit: true
-    config.add_facet_field 'era_facet', label: 'Era'  
+    config.add_facet_field 'era_facet', label: 'Era'
     config.add_facet_field 'author_other_facet', label: 'Organization (as author)', limit: true
+    config.add_facet_field 'donor_tags_sim', label: 'Donor tags', limit: true
```

Comment:
I might like a comment in here or somewhere to indicate
1.  it only applies to Feigenbaum as of 2015/10
2.  perhaps this shouldn't display for ALL exhibits -- i think it will be noisy for curators (and they might think "hey, can't I have donor tags)
3.  in the future we may refactor so that single exhibit specific configuration is possible